packages/SwingSet/docs/virtual-objects.md

```diff
@@ -80,9 +75,11 @@ and then use it like this:
   console.log(`${barCounter.getName()} count is ${barCounter.getCount()`); // "new bar count is 1"
 ```
 
+Note that the `init` and `actualize` functions are defined explicitly here for clarity of exposition, but in practice you'd usually declare them inline as arrow functions in the parameters of the `defineKind` call.
+
 Additional important details:
 
-- The set of state properties is determined by the `init` function.  The properties that exist when `init` returns _are_ the properties of the object.  State properties cannot thereafter be added or removed.
+- The set of state properties is fully determined by the `init` function.  The of the object that `init` returns _are_ the properties of the virtual object's state.  State properties cannot thereafter be added or removed.
```

Comment:
typo, "The of the object".. maybe say something more like:

The set of properties on `state` are completely determined by the enumerable properties of the object that `init` returns. After initialization, the values of `state`'s properties can be modifed, but new properties cannot be added, nor existing ones removed.